types/react-form/index.d.ts

```diff
@@ -1,36 +1,25 @@
-// Type definitions for react-form 1.3
+// Type definitions for react-form 2.12
```

Comment:
Please back up the old typings to a folder called v1 https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version